internal/util/importutilv2/option.go

```diff
@@ -113,3 +113,12 @@ func GetCSVSep(options Options) (rune, error) {
 	}
 	return []rune(sep)[0], nil
 }
+
+func GetCSVNullKey(options Options) (string, error) {
+	nullKey, err := funcutil.GetAttrByKeyFromRepeatedKV("nullkey", options)
+	defaultNullKey := ""
```

Comment:
seems no need to set a 'defaultNullKey', because reader only try to parse null when nullable. Can simply just return 'NULL' as  the null key.

Reply:
This function is used to parse the nullkey configured by the user. In addition to the defaultNullKey, the user can configure other nullkeys to increase the flexibility of the export format. For example, the NULL data exported by pg supports configuration.
If it is not necessary, I can directly return "NULL".

Reply:
> This function is used to parse the nullkey configured by the user. In addition to the defaultNullKey, the user can configure other nullkeys to increase the flexibility of the export format.
> For example, the NULL data exported by pg supports configuration http://www.postgres.cn/docs/13/sql-copy.html.
> If it is not necessary, I can directly return "NULL".

I see. it's ok for me now to follow the pg support configuration.